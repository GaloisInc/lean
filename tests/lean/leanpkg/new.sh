rm -rf new
leanpkg new new
ls -a new
rm -rf new
