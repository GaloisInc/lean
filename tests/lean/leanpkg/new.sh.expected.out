> mkdir -p new
> cd new
> mkdir src
> git init -q
configuring new 0.1
.
..
.git
.gitignore
leanpkg.path
leanpkg.toml
src
0
